### PR TITLE
Add support for modifying docdb parameters

### PR DESCRIPTION
**Pull request by tenyo**
The modify request can be used to change the master password for the DocumentDB cluster, or other parameters, such as `BackupRetentionPeriod`, `EngineVersion` or `DBInstanceClass`

### CODE REVIEW

**Review by FishNecks — COMMENTED** [review]
Does it make more sense to have the user only pass the field they want changed and assume others stay unchanged?

**Comment by tenyo** [comment]
> Does it make more sense to have the user only pass the field they want changed and assume others stay unchanged?

Yes, you can pass one or multiple ones. In the example I wanted to show changing several at a time, but you can just pass a single one.

**Review by fishnix — APPROVED** [review]
lgtm 👍

File: README.md [hunk 1]
```diff
@@ -272,6 +273,158 @@ GET `/v1/docdb/{account}/{name}`
...
+
+The modify request can be used to change the master password for the DocumentDB cluster, or other parameters, such as `BackupRetentionPeriod`, `EngineVersion` or `DBInstanceClass`
+
+PUT `/v1/docdb/{account}/{name}`
+
+```json
+{
+  "BackupRetentionPeriod": 2,
+  "DBInstanceClass": "db.r5.large",
+  "MasterUserPassword": "newexamplepassword",
```
Comment (FishNecks) [inline README.md]: I'm not sure this is a valid body with the trailing comma.

**Pull request merged**